### PR TITLE
Sleep fixes to avoid premature actions

### LINKED ISSUE #24: When docking up it tries to unload cargo too early, before docked properly
I noticed the bot tries to unload cargo too early, after the bot did a couple of runs and went home after cargo hold was full. Could be random, but we got to make sure to sleep longer after clicking dock in the circle menu. I think this happens because I removed the two steps with warping and docking. Now it just docks. So I would need to sleep for two times 70 seconds I guess.

@x-actly ☝️

### PR DESCRIPTION
**Pull request by jarlah**
fixes #24

### CODE REVIEW

File: functions.py [hunk 1]
```diff
@@ -54,17 +44,19 @@ def click_circle_menu(x, y, x_offset, y_offset):
...
     x_offset = -50
     y_offset = random.randint(-51, -49)
     click_circle_menu(x, y, x_offset, y_offset)
 
-def click_top_center_circle_menu(x, y):
+def click_dock_circle_menu(x, y):
     x_offset = 0
     y_offset = random.randint(-51, -49)
     click_circle_menu(x, y, x_offset, y_offset)
+    # sleep longer when clicking the docking button in the circle menu
```
Comment (jarlah) [inline functions.py:58]: this is the fix

**Comment by jarlah** [comment]
testing it currently

File: functions.py [hunk 2]
```diff
@@ -82,30 +74,14 @@ def drone_out(x,y):
...
     log("drones returning to bay...")
     # drone in
     pyautogui.keyDown('shift')
     pyautogui.press('r')
     sleep_and_log(1)
     pyautogui.keyUp('shift')
     # drone time back to ship
-    sleep_and_log(random_sleep_small)
-
-def docking_dropdown(x, y, rel_x, rel_y):
```
Comment (jarlah) [inline functions.py:95]: no longer need for this since we click on dock in circle menu

File: functions.py (hunk 1 above)
Comment (jarlah) [reply functions.py:58]: a little bit too long sleep, but its better than too little sleep

**Review by x-actly — APPROVED** [review]

**Pull request merged**